checks/external_calls.yml

```diff
@@ -1,5 +1,5 @@
 name: php_external_calls
-command: 'sudo lsof -a -iTCP -sTCP:ESTABLISHED -cphp -P -Fn | grep -v -E "^p|:3306\$|:11211\$|^f[0-9]{1,2}\$\"'
```

Comment:
The removal of the TCP state makes sense here.  What is the change in the grep for?

Reply:
That quotation to end the grep doesn't need to be escaped I think. We're getting all those file descriptor entries in tickets even though they're being grepped out in `^f[0-9]{1,2}`.